application/android/src/co/theengine/loomplayer/LoomSensors.java

```diff
@@ -192,14 +187,15 @@ public void onSensorChanged(SensorEvent event)
                     ///if the sensor data is unreliable return
                     if(event.accuracy == SensorManager.SENSOR_STATUS_UNRELIABLE)
                     {
-                        return;
+                     //   return;
                     }
 
                     ///get correctly oriented values to work with
                     remappedValues = remapXYZValues(event.values);
-                    x = remappedValues[0];
-                    y = remappedValues[1];
-                    z = remappedValues[2];
+                    //normalize the values
+                    x = - remappedValues[0] / 10;
+                    y = - remappedValues[1] / 10;
+                    z = - remappedValues[2] / 10;
```

Comment:
Do you have a source for that -10, 10 figure? I can only find this SO answer that talks about the range a bit: http://stackoverflow.com/a/19981219

I think that we should figure out what kind of a value Android and iOS even report (m/s^2?) and then decide on the one we want, define it in the docs and transform to it if needed. I think it's more meaningful that way than some arbitrarily transformed number.